package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "chart.js": "^2.9.4",
-    "eslint": "^7.9.0",
+    "eslint": "^7.19.0",
```

Comment:
It seems like `docker exec tcf_django npm install eslint` updated the versions of ESLint dependencies in `package.json` and `package-lock.json`. (`docker exec tcf_django npm install`--or `i` instead of `install` if you want a shorter command--should have been enough.) Since updating dependencies is outside the scope of this PR, could you revert those changes? Don't worry about commit history because this PR will be squash-merged anyway.

Reply:
Gotcha! The unintended stuff should have been reverted now?